src/utils/url.ts

```diff
@@ -40,15 +41,25 @@ export const openRoomUrl = (url: string) => {
 };
 
 export const openUrl = (url: string) => {
-  isValidUrl(url)
-    ? window.open(url, "_blank", "noopener,noreferrer")
-    : console.error(
-        `Invalid URL ${url} on page ${window.location.href}; ignoring`
-      );
+  if (!isValidUrl(url)) {
+    Bugsnag.notify(
+      new Error(`Invalid URL ${url} on page ${window.location.href}; ignoring`),
+      (event) => {
+        event.addMetadata("utils/url::openUrl", { url });
+      }
+    );
+    return;
+  }
+
+  window.open(url, "_blank", "noopener,noreferrer");
 };
 
-export const isValidUrl = (url: string) => {
-  return VALID_URL_PROTOCOLS.includes(new URL(url).protocol);
+export const isValidUrl = (url: string): boolean => {
+  try {
+    return VALID_URL_PROTOCOLS.includes(new URL(url).protocol);
+  } catch (e) {
```

Comment:
nit: consider catching only `TypeError` here: https://developer.mozilla.org/en-US/docs/Web/API/URL/URL

Reply:
resolved in https://github.com/sparkletown/sparkle/pull/889/commits/935aeb69024ccebf57b50e64566a4edd511425d7